src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -89,6 +114,102 @@ export const HookPasswordValidatedInput = ({
   );
 };
 
+export const ValidatedInputAndTextArea = ({
+  value,
+  stepValidation,
+  fieldName,
+  onBlur,
+  placeholder,
+  onChange,
+  ariaLabel,
+  inputType = 'textInput',
+}: ValidationInputProp) => {
+  const errorMessage = stepValidation.errors[fieldName];
+  const hasError = errorMessage !== '';
+
+  return (
+    <>
+      <TextAndTextAreaInput
+        inputType={inputType}
+        ariaLabel={ariaLabel || ''}
+        value={value}
+        placeholder={placeholder || ''}
+        errorMessage={errorMessage}
+        onChange={onChange!}
```

Comment:
Why do we need a non-null assertion operator here?

Reply:
because it detects that the properties onChange and onBlur might be undefined at runtime, even though the code seems to assume they are always defined. The reason is that they are defined as Optional props.
and we know that it is not optional in our case, so I will define it as mandatory 
```
   onChange: (
      event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>,
      value: string
    ) => void;
```

thank you for that